business/openid_auth.go

```diff
@@ -1,19 +1,29 @@
 package business
 
 import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/sha256"
 	"crypto/tls"
+	"encoding/base64"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/url"
 	"strings"
 	"time"
 
+	"github.com/dgrijalva/jwt-go"
```

Comment:
trivial - a newline should be on the next line to separate the third-party deps from the kiali deps.